deploy/docker/Dockerfile-kubernetes-tool

```diff
@@ -8,12 +8,30 @@ RUN go build -o tools/kubernetes/kuiper-kubernetes-tool tools/kubernetes/main.go
 
 FROM alpine:3.12
 
-WORKDIR /kuiper-kubernetes-tool
+ENV KUIPER_HOME="/kuiper-kubernetes-tool"
+
+ARG KUIPER_USER="kuiper"
+ARG KUIPER_USER_ID="1001"
+
+# Run the following under the root user for proper build
+RUN apk add sed libzmq && \
+    adduser -DH -s /sbin/nologin -u ${KUIPER_USER_ID} ${KUIPER_USER}
+
+WORKDIR ${KUIPER_HOME}
 
 RUN mkdir -p log
 
 COPY --from=builder /go/kuiper/tools/kubernetes/kuiper-kubernetes-tool .
 COPY --from=builder /go/kuiper/tools/kubernetes/etc etc
 COPY --from=builder /go/kuiper/tools/kubernetes/sample sample
 
+# Set appropriate ownership to allow binary full access to /kuiper dir
+RUN chown -Rh ${KUIPER_USER}:${KUIPER_USER} ${KUIPER_HOME}
+
+# Create taos db directory and set kuiper to owner
+RUN mkdir -p /usr/local/taos && chown -Rh ${KUIPER_USER}:${KUIPER_USER} /usr/local/taos
+
```

Comment:
I think `adduser` and `chown` can be combined in the same `RUN`, which makes it easier to read, don't you think?

Reply:
No, they are separated because of build timing - "root" has to create the user and perform build processes, and then assign ownership after the build processes are complete.

Reply:
This has been updated as well.